far/FarPol.hlf.m4

```diff
@@ -3054,18 +3053,19 @@ Za: [L. Jakubowicz; "Wiadomości Literackie" (nr 1, 1936, str. 7)]
  Strona kodowa 65001 (UTF-8)
 
 @-
- \1b0000000035: D0 92 20 D1 87 D0 B0 D1 │ 89 D0 B0 D1 85 20 D1 8E  В› ч›а›щ›а›х› ю›\-
- \1b0000000045: D0 B3 D0 B0 20 D0 B6 D0 │ B8 D0 BB 2D D0 B1 D1 8B  г›а› ж›и›л›-б›ы›\-
- \1b0000000055: D0 BB 20 D1 86 D0 B8 D1 │ 82 D1 80 D1 83 D1 81 2C  л› ц›и›т›р›у›с›,\-
+ \1b00000000C0: 57 20 7A 61 72 6F C5 9B │ 6C 61 63 68 20 70 6F C5  W zaroś›lach poł\-
```

Comment:
This sentence is a pangram, i.e. it contains all the alphabet letters.
If you want to translate it, it might be better to pick an existing Polish one, e.g.
`Pójdźże, kiń tę chmurność w głąb flaszy`